test/acquisition/test_knowledge_gradient.py

```diff
@@ -3,7 +3,7 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-
+from contextlib import ExitStack
```

Comment:
Mind adding an empty line here? Our linter is complaining about this. 

```suggestion

from contextlib import ExitStack
```